noruba/features/lib/step_definitions/noruba_steps.rb

```diff
@@ -156,6 +66,22 @@ def make_arg_list(args)
   write_file('cucumber.yml', profile)
 end
 
+Given('log only formatter is declared') do
+  write_file('features/support/log_only_formatter.rb', [
+    'class LogOnlyFormatter',
+    '  attr_reader :io',
+    '',
+    '  def initialize(config)',
+    '    @io = config.out_stream',
+    '  end',
+    '',
+    '  def attach(src, media_type)',
+    '    @io.puts src',
+    '  end',
+    'end'
+].join("\n"))
```

Comment:
Whitespace is wonky here. (Thanks for the hard work on this!)

```suggestion
  ].join("\n"))
```

Reply:
indeed thanks for checking :) I haven't even tried yet to get Rubocop passing on this huge and ugly step definition file, but I guess I'll spend a few hours on cleanup if I ever get to ee all scenarios passing.
The good news so far is that I have the same failing tests on Windows and the various Ruby versions. So we may end up with a stable build on Windows  (which would be great)